lib/supa/commands/attribute.rb

```diff
@@ -2,8 +2,20 @@ module Supa
   module Commands
     class Attribute < Supa::Command
       def represent
+        return if hide?
+
         @tree[@name] = value
       end
+
+      private
+
+      def hide?
+        value.nil? && hide_when_nil?
+      end
+
+      def hide_when_nil?
```

Comment:
Not sure if I understood it correctly. Do you want to use `hide_when_empty?` here? If yes then done https://github.com/distribusion/supa/pull/20/commits/2bfe9b10d8aa020c5f363390a9c9bb205dee386b.